src/app/[locale]/specs/account/page.mdx

```diff
@@ -0,0 +1,90 @@
+
+## Hosting Status
+
+On any network service, a known account is either `active` or not (a boolean flag) at a given point in time. If an account is not active, the service should not redistribute content for that account (repositories, individual records, blobs, etc). If the account is unknown (never seen before), the state can be undefined.
+
+Identity metadata (DID documents and handle status), and the account hosting status itself, are distinct from hosted content, and can be redistributed.
+
+Network services are encouraged to implement API endpoints (such as `com.atproto.sync.getRepoStatus`) which describe current hosting status for individual accounts. If they expose an event stream, they should also emit `#account` events when their local account hosting status updates for an account.
+
+In addition to the `active` boolean, an account might in a more specific state (all of which correspond with `active=false`):
+
+- `deleted`: user or host has deleted the account, and content should be removed from the network. Implied permanent or long-term, though may be reverted (deleted accounts may reactivate on the same or another host).
+- `deactivated`: user has temporarily paused their overall account. Content should not be displayed or redistributed, but does not need to be deleted from infrastructure. Implied time-limited. Also the initial state for an account after migrating to another PDS instance.
+- `takendown`: host or service has takendown the account. Implied permanent or long-term, though may be reverted.
+- `suspended`: host or service has temporarily paused the account. Implied time-limited.
+
+New account states will be added in the future. To prevent broken expectations, relevant API endpoints (such as `com.atproto.sync.getRepoStatus`, and the `#account` event on the `com.atproto.sync.subscribeRepos` event stream) break out `active` as a boolean flag, and then clarify non-active accounts with a separate `status` string. Services should use the `active` flag to control overall account visibility (observable behavior) with the `status` string acting as clarification which might determine more specific infrastructure behaviors (such as data deletion).
+
+The deactivation and suspension states are implied to be temporary, though they might have indefinite time periods. Deletion and takedowns are implied to be more final, but can still technically be reverted (and frequently are, in practice).
+
+Downstream services might decide to delete content rapidly upon takedown (like deletion), or simply stop redistributing content until a later time.
+
+Identities might also be tombstoned, which is also implied to be permanent and final, but can technically be reversed in certain conditions. If the account's identity is `tombstoned`, the account hosting status should be interpreted as `deleted`.
```

Comment:
should `tombstoned` be in the list above? or are tombstoned a subset of "status==`deleted`" accounts differentiated at some other layer?  "interpreted as" is a little confusing here, couldn't hurt to be more explicit about what's same-as and what's different, particularly if `tombstoned` is a member of the enum above.

Reply:
This gets at the distinction between "identity" and "account" in atproto, which I tried to sketch out in the intro paragraph of this doc.

An identity is a DID, and DIDs have a concept of "tombstoned". But accounts are held on a PDS instance, and those accounts don't have a lifecycle state called "tombstoned", and it is totally possible for the DID to be tombstoned and the PDS isn't aware of it (eg, a `did:web` can be updated without notifying the PDS).

This paragraph is talking about how third party services could/should reconcile possibly contradicting states between identity and account.